app/dashboard/utils.py

```diff
@@ -172,6 +172,11 @@ def get_bounty(bounty_enum, network):
     if 'Failed to get block' in bounty_data_str:
         raise IPFSCantConnectException("Failed to connect to IPFS")
 
+    # https://github.com/Bounties-Network/StandardBounties/issues/25
+    override_deadline = bounty_data.get('payload', {}).get('expire_date', False)
```

Comment:
Will the override ever be `False`?

Reply:
yes.. for bounties that were submitted on the bounties network or on gitcoin, but before this PR is merged